app/views/deploys/index.html.erb

```diff
@@ -9,12 +9,17 @@
   <div class="row">
     <%= search_form do %>
       <div class="col-sm-2">
-        <%= label_tag :deployer %>
-        <%= text_field_tag 'search[deployer]', params.dig(:search, :deployer), placeholder: 'deployer name', class: "form-control" %>
+        <%= label_tag 'Deployer name' %>
+        <%= text_field_tag 'search[deployer]', params.dig(:search, :deployer), class: "form-control" %>
       </div>
 
-      <% unless @project %>
-        <div class="col-sm-2">
+      <% if @project %>
+        <div class="col-sm-1">
+          <%= label_tag :git_sha %>
+          <%= text_field_tag 'search[git_sha]', params.dig(:search, :git_sha), class: "form-control" %>
+        </div>
+      <% else %>
+        <div class="col-sm-1">
```

Comment:
sm-2 was breaking the line ... so had to make that smaller ... maybe we need to have a smarter way for cases with too many options ...